EssentialsDiscord/README.md

```diff
@@ -0,0 +1,463 @@
+# EssentialsX Discord
+
+EssentialsX Discord is a module that brings a simple, lightweight, easy-to-use, and bloat-free
+bridge between Discord and Minecraft.
+
+EssentialsX Discord offers *essential* features you'd want from a Discord bridge such as:
+* MC Chat -> Discord Channel
+* Discord Channel -> MC Chat
+* Basic MC -> Discord Event Monitoring (Join/Leave/Death/Mute)
+* MC Console -> Discord Relay
+* Discord Slash Commands 
+  * /execute - Execute console commands from Discord
+  * /msg - Message Minecraft players from Discord
+  * /list - List players currently online on Minecraft
+* & more...
+
+---
+
+## Table of Contents
+> * [Initial Setup](#initial-setup)
+> * [Console Relay](#console-relay)
+> * [Configuring Messages](#configuring-messages)
+> * [Receive Discord Messages in Minecraft](#receive-discord-messages-in-minecraft)
+> * [Discord Commands](#discord-commands)
+> * [Misc Permissions](#misc-permissions)
+> * [Developer API](#developer-api)
+
+---
+
+## Initial Setup
+
+0. Before starting your server, there are a few steps you have to take. First, you must create a new
+discord bot at [discord.com/developers/applications](https://discord.com/developers/applications/).
+
+1. Once on that page, click on "New Application" button on the top right, give your bot a name, and
+then click "Create".
+> ![Creating Application](https://i.imgur.com/8ffp4R1.gif)
+> `New Application` -> Give Application a Name -> `Create`
+
+2. Once you create the application, you'll be directed to its overview. From this screen, you'll
+need to copy your "Client ID" and save it for a later step. To copy your client id, click the
+upper-left most blue "Copy" button. Make sure to save it for a later step.
+> ![Copy Client ID](https://i.imgur.com/W3OMTu5.gif)
```

Comment:
it's called client id on the page, which is why i called it that here

Reply:
When I was going through the docs again to do a fresh installation I noticed they changed it. It's now called Application ID.

Reply:
They are the same thing :p. In the "General information" tab, it's called Application ID. In the "OAuth2" tab, it's called Client ID

Reply:
I for my part would stick with Client ID, since you add (or "invite") the Client account of a bot into your server and don't really install an application or something.

Reply:
I recommend updating the GIF and using Application ID found on the General Information tab, as that requires less effort from the end-user and is more intuitive
*also "ID" needs to be fully caps*

Reply:
> I recommend updating the GIF and using Application ID found on the General Information tab, as that requires less effort from the end-user and is more intuitive

Not worth it, button is still in the same position

Reply:
I'm pretty indifferent to what we're calling it, but the idea was simply to update the wording so it matches what is seen on the site, to avoid potential confusion while installing.

Reply:
Button's still in the same place as JRoy mentioned so it isn't really a big problem, and you can figure it out from the screenshot.